src/xunit.performance.api/Profilers/Etw/EtwManagedModule.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Xunit.Performance.Api.Profilers.Etw
+{
+    public sealed class EtwManagedModule
```

Comment:
If I remember correctly from our conversation, you were going to put this behind an interface.  If so, you should make these classes internal and just have a public interface.  Also, I think that you probably want ManagedModule to inherit from Module since its set of properties is strictly a superset of Module.

Reply:
Yes, I'll be pushing these changes in next commit :)

---
In reply to: [143034762](https://github.com/Microsoft/xunit-performance/pull/241#discussion_r143034762) [](ancestors = 143034762)